collection-scripts/gather_ceph_resources

```diff
@@ -374,7 +374,20 @@ for ns in $namespaces; do
                     timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd mirror pool info $bp --format=json --pretty-format" 2>>"${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-mirror-pool-info-"${bp}"-debug.log
                 } >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_pool_info &
                 pids_rbd+=($!)
-                printf "Collecting mirror image status for images in: %s\n" "${bp}" >>"${COMMAND_OUTPUT_DIR}"/rbd_mirror_image_status
+                # Collecting mirror image status for provider mode with rados namespace
```

Comment:
This should be done for `rbd mirror pool status --verbose` and `rbd mirror pool info` as well, not just for `rbd mirror image status`:

> "rbd mirror pool info",  "rbd mirror pool status" and "rbd mirror image status" need to be applied to each RBD namespace the same way they are applied to a pool (i.e. the default namespace).

Reply:
As mentioned in jira, rbd mirror pool status and info shows the same output with or without the namespace, I dont think this makes any difference, please correct me if I am wrong

Reply:
> rbd mirror pool status and info shows the same output with or without the namespace

It shouldn't be the same -- in fact a bug in "rbd mirror pool status" output for namespaces was just discovered and is being fixed under https://tracker.ceph.com/issues/69911.